.yaydoc.yml

```diff
@@ -0,0 +1,12 @@
+metadata:
+  author: FOSSASIA
+  projectname: Phimp.me
+  version: development
+build:
+  theme: sphinx_fossasia_theme
+  source: docs
+  logo: _static/logo.png
```

Comment:
Since the logo is already being displayed in the README, I guess we don't really need this.